centreon/tests/php/Centreon/Domain/Common/AssertionTest.php

```diff
@@ -260,6 +260,9 @@ function () use ($propertyPath, $failAssertion, $failMessage, $failCode): void {
     'inArray(NULL, [1, "a", NULL])' => [
         fn() => Assertion::inArray(null, [1, 'a', null], $propertyPath),
     ],
+    'range(1.1, 1.0, 2)' => [
```

Comment:
It lacks the failure tests in the end of the file.

Reply:
@callapa